migrations/codeintel/1676423214_remove_lsif_data/down.sql

```diff
@@ -0,0 +1,163 @@
+COMMENT ON COLUMN lsif_data_references_schema_versions.min_schema_version IS 'A lower-bound on the `lsif_data_references.schema_version` where `lsif_data_references.dump_id = dump_id`.';
+COMMENT ON COLUMN lsif_data_references_schema_versions.max_schema_version IS 'An upper-bound on the `lsif_data_references.schema_version` where `lsif_data_references.dump_id = dump_id`.';
+
+CREATE TABLE IF NOT EXISTS lsif_data_result_chunks (
+    dump_id integer NOT NULL,
+    idx integer NOT NULL,
+    data bytea,
+    PRIMARY KEY (dump_id, idx)
+);
+
+COMMENT ON TABLE lsif_data_result_chunks IS 'Associates result set identifiers with the (document path, range identifier) pairs that compose the set.';
+COMMENT ON COLUMN lsif_data_result_chunks.dump_id IS 'The identifier of the associated dump in the lsif_uploads table (state=completed).';
+COMMENT ON COLUMN lsif_data_result_chunks.idx IS 'The unique result chunk index within the associated dump. Every result set identifier present should hash to this index (modulo lsif_data_metadata.num_result_chunks).';
+COMMENT ON COLUMN lsif_data_result_chunks.data IS 'A gob-encoded payload conforming to the [ResultChunkData](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@3.26/-/blob/enterprise/lib/codeintel/semantic/types.go#L76:6) type.';
+
+CREATE INDEX IF NOT EXISTS lsif_data_definitions_dump_id_schema_version ON lsif_data_definitions USING btree (dump_id, schema_version);
+CREATE INDEX IF NOT EXISTS lsif_data_definitions_schema_versions_dump_id_schema_version_bo ON lsif_data_definitions_schema_versions USING btree (dump_id, min_schema_version, max_schema_version);
+CREATE INDEX IF NOT EXISTS lsif_data_documents_dump_id_schema_version ON lsif_data_documents USING btree (dump_id, schema_version);
+CREATE INDEX IF NOT EXISTS lsif_data_documents_schema_versions_dump_id_schema_version_boun ON lsif_data_documents_schema_versions USING btree (dump_id, min_schema_version, max_schema_version);
+CREATE INDEX IF NOT EXISTS lsif_data_implementations_dump_id_schema_version ON lsif_data_implementations USING btree (dump_id, schema_version);
+CREATE INDEX IF NOT EXISTS lsif_data_implementations_schema_versions_dump_id_schema_versio ON lsif_data_implementations_schema_versions USING btree (dump_id, min_schema_version, max_schema_version);
+CREATE INDEX IF NOT EXISTS lsif_data_references_dump_id_schema_version ON lsif_data_references USING btree (dump_id, schema_version);
+CREATE INDEX IF NOT EXISTS lsif_data_references_schema_versions_dump_id_schema_version_bou ON lsif_data_references_schema_versions USING btree (dump_id, min_schema_version, max_schema_version);
+
+CREATE OR REPLACE TRIGGER lsif_data_definitions_schema_versions_insert AFTER INSERT ON lsif_data_definitions REFERENCING NEW TABLE AS newtab FOR EACH STATEMENT EXECUTE FUNCTION update_lsif_data_definitions_schema_versions_insert();
```

Comment:
databases cap says no 
```suggestion
DROP TRIGGER lsif_data_definitions_schema_versions_insert ON lsif_data_definitions;
CREATE TRIGGER lsif_data_definitions_schema_versions_insert AFTER INSERT ON lsif_data_definitions REFERENCING NEW TABLE AS newtab FOR EACH STATEMENT EXECUTE FUNCTION update_lsif_data_definitions_schema_versions_insert();
```

Reply:
[bcd7434](https://github.com/sourcegraph/sourcegraph/pull/46127/commits/bcd743440227e335cb08c22c7872d0d36523cc15)